content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,452 @@
+
+The author has included, directly in darktable, a Lua installer. This can be enabled or disabled on first boot or in the preferences. The installer downloads the latest available scripts from the darktable Lua scriptsrepository.
+
+Of course, this is only for those who don't already have the manager script installed. This is what this new installer looks like:
+
+  <div style="text-align: center">
+![lua scripts installer]({filename}en/lua-scripts-installer.png)
+  </div>
+
+Once installed, script manager appears on the left panel in lighttable:
+  <div style="text-align: center">
+![script manager]({filename}en/script-manager.png)
+  </div>
+
+## An adjustable histogram and a new mode: RGB parade
+
+Two new features in darktable 3.2 histogram. First is the possibility to adjust histogram height size. This adjustment is done the same way as some lighttable modules: move the cursor over the histogram and press `Ctrl` + `scroll` to adjust the height.
+
+A new mode, RGB parade, appears and displays waveforms that represent the levels of each layer (red, green and blue). This mode allows you to visualize the distribution of color components.
+
+This mode can also be displayed by selecting the histogram display in wave mode (with the top left icon on the histogram) and then  changing the wave mode with the next icon on the right.
+
+  <div style="text-align: center">
+![RGB parade mode]({filename}en/rgb-parade.png)
+  </div>
+
+Add to that an important code rewriting of the histogram for better performances.
+
+## Drawn mask: curved gradient
+
+When adding a drawn mask, it is now possible to bend the gradient shape (gradient). To do this, point the mouse cursor at the gradient bar and scroll up or down with the mouse wheel (or touchpad). `Ctrl` + `scroll` allows you to set the opacity and `Shift` + `scroll` manages the edge hardness.
+
+## An improved history
+
+The history behaviour has been improved according to different users feedback. When compressing the history or loading a new image, the history will now behave this way:
+
+- any active module that cannot be deactivated will always appear first in the history stack;
+- any module that is enabled by default but can be disabled appears in the history stack;
+- finally, all modules activated by presets or as a result of a preference setting (workflow or sharpness selected in the preferences) are added.
+
+## And more:
+
+- Add support of AVIF file format (>= 0.7). This require the [libavif](https://github.com/AOMediaCodec/libavif) library to be installed.
+- Add optional grey-scale export of TIFF for monochrome images.
+- It is now possible to export masks in TIFF format.
+- It is now possible to use more than 500 images in tethered control, which is needed for time-lapse.
+- Applying a style now supports _append_ mode (added to the existing image history) or in _overwrite_ mode (so replace the existing image history). This makes the style module consistent with _development_ module.
```

Comment:
"..._append_ mode (**to add the selected style** to the existing image history) **and** _overwrite_ mode (**to replace the existing image history with the selected style**)"
"...consistent with **the _history stack_** module."